### DISCUSSION TITLE
Make host and key settable e.g. via setup.nsi

### DISCUSSION THREAD

**Discussion by RipperFox** [rustdesk/rustdesk#1115] [General]
While it's possible to set host & key via the filename of the executable, it would be nice to have a way to deploy those settings via the installer and leave the filename untouched - would also ease e.g. later automatic updates.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/blob/master/setup.nsi help youself.

**Comment by RipperFox** [comment]
Helped myself..
Thanks for the hint that I have to modify/deploy a cutom %appdata%\RustDesk\config\RustDesk2.toml  
Btw: What's the difference there between "rendezvous_server" and "custom-rendezvous-server"?
